cmd/world/main.go

```diff
@@ -1,19 +1,38 @@
 package main
 
 import (
-	"github.com/getsentry/sentry-go"
+	"github.com/denisbrodbeck/machineid"
+	ph "github.com/posthog/posthog-go"
 	"log"
+	"os"
+	"pkg.world.dev/world-cli/common/logger"
+	"pkg.world.dev/world-cli/posthog"
 	"time"
 
+	"github.com/getsentry/sentry-go"
 	"pkg.world.dev/world-cli/cmd/world/root"
 )
 
+var (
+	AppVersion    string
+	PosthogApiKey string
```

Comment:
where does this get set?

Reply:
From -ldflags param when building apps
Example : `go build -ldflags "-X main.AppVersion={appVersion} -X main.SentryDsn={sentryDsn} -X main.PosthogApiKey={apiKey}`

Reply:
doh! i shoulda read the PR description 🫠

Reply:
:smile: 
Any other concern about this @technicallyty? 

The purpose is to set keys (sentry and posthog) using github secret key in ci/cd (prevent key leak), and also automatically set the app version using release version.

Reply:
let's just add some comments in the code so contributors can understand how/when those variables get set.

Reply:
updated